packages/svelte/src/internal/client/dev/ownership.js

```diff
@@ -72,5 +72,9 @@ function is_bound(props, prop_name) {
 	// Can be the case when someone does `mount(Component, props)` with `let props = $state({...})`
 	// or `createClassComponent(Component, props)`
 	const is_entry_props = STATE_SYMBOL in props || LEGACY_PROPS in props;
-	return !!get_descriptor(props, prop_name)?.set || (is_entry_props && prop_name in props);
+	return (
+		!!get_descriptor(props, prop_name)?.set ||
+		(is_entry_props && prop_name in props) ||
+		!(prop_name in props)
```

Comment:
While that's all correct the function name now is confusing because it no longer is what this check is about. `is_bound_or_unset` maybe?

Reply:
Mmm yeah let me change that, even tho I read that as "if it's unset is automatically bound" but maybe it's better to rename it